ovos_config/mycroft.conf

```diff
@@ -578,17 +559,17 @@
   // Override: REMOTE
   "stt": {
     // select a STT plugin as described in the respective readme
+    // ovos-stt-plugin-server default plugin has a bundled list of public whisper instances
     "module": "ovos-stt-plugin-server",
-    "fallback_module": "",
-    // the default instance is hosted by a OpenVoiceOS member
-    // it is a google proxy equivalent to mycroft selene
-    "ovos-stt-plugin-server": {"url": "https://stt.openvoiceos.com/stt"}
+    // set a offline fallback STT, if your primary STT fails for some reason
+    "fallback_module": ""
   },
 
   // Text to Speech parameters
   // Override: REMOTE
   "tts": {
     "pulse_duck": false,
+    // TODO - ovos tts server plugin public servers list by default
```

Comment:
I think this can be changed to `ovos-tts-plugin-server` with the default public servers now.